lib/clients/websocket.js

```diff
@@ -8,17 +8,21 @@ var signRequest = require('../../lib/request_signer').signRequest;
 /**
  * Create a new connection to a websocket feed
  * @param productIDs {array} The GDAX products to listen to. Default: ['BTC-USD']
- * @param websocketURI {string} Optional. The websocker URL. Default: The official GDAX feed.
- * @param auth {object} Optional. An object containing your API ket details (key, secret & passphrase)
+ * @param options {object} Optional. It can contain auth, websocketURI and heartbeat
+ *  - auth {object} Optional. An object containing your API ket details (key, secret & passphrase)
+ *  - websocketURI {string} Optional. The websocker URL. Default: The official GDAX feed.
+ *  - heartbeat {boolean} Optional. Receive heartbeat messages once a second
  */
-var WebsocketClient = function(productIDs, websocketURI, auth) {
+var WebsocketClient = function(productIDs, options) {
   var self = this;
+  options = options || {};
```

Comment:
Due to the breaking changes in the way the `WebsocketClient` takes parameters, do you think it would be good to add the following more parameter error handling?

For example, maybe checking to ensure that `options` is in fact an object, and not a string (perhaps websocketURI).